modules/serverless-build-func-kn.adoc

```diff
@@ -6,49 +6,42 @@
 [id="serverless-build-func-kn_{context}"]
 = Building functions
 
-Before you can run a function, you must build the function project by using the `kn func build` command. The build command reads the `func.yaml` file from the function project directory to determine the image name and registry.
+Before you can run a function, you must build the function project by using the `kn func build` command. The build command reads the `func.yaml` file from the function project directory to determine the image name and the registry used to store functions images.
 
 .Example `func.yaml`
 [source,yaml]
 ----
 name: example-function
 namespace: default
 runtime: node
-image: <image_from_registry>
+image: image-registry.openshift-image-registry.svc:5000/<oc_project>/<function_name>
 imageDigest: ""
 trigger: http
 builder: default
 builderMap:
-  default: quay.io/boson/faas-nodejs-builder
+  default:
 envs: {}
 ----
 
-If the image name and registry are not set in the `func.yaml` file, you must either specify the registry flag, `-r` when using the `kn func build` command, or you are prompted to provide a registry value in the terminal when building a function. An image name is then derived from the registry value that you have provided.
+By default, the OpenShift Container Registry is used in `func.yaml` when you create a new project. The registry used to store function images can be overridden by using the `-r` flag or the `FUNC_REGISTRY` environment variable.
 
-.Example command using the `-r` registry flag
+.Example build command
 [source,terminal]
 ----
-$ kn func build [-i <image> -r <registry> -p <path>]
+$ kn func build [-i <image> -p <path>]
```

Comment:
`func build` now has a lot of flags. I wonder if it makes sense to show any here.  I would suggest removing all flags from the example, or having some text such as "To learn more about the build command options, use the help command `kn func help build`".